docs/guide/databases.md

```diff
@@ -17,8 +17,9 @@ and enable `DoctrineSchemaBundle` in `AppKernel.php`.
 
 When you run `composer install`, you will be asked to [provide installation parameters](../getting_started/install_ez_platform.md#provide-installation-parameters).
 
-If you use PostgreSQL, two parameters need to be set differently than when using MySQL:
+If you use PostgreSQL, the following parameters need to be set differently than when using MySQL:
 
+- `env(DATABASE_PLATFORM)` must be set to `pgsql` instead of `mysql`
```

Comment:
This is ok-ish, but actually outdated. It applies to `parameters.yml` config from v3. Right now we use `.env` (see the changes in the other file), so we should align this doc.

Offtopic: about `.env`, this file is committed to a project repository, so we should not encourage to store database credentials there. Those should be always stored in unversioned local files, like `.env.local` or `.env.dev.local` or `.env.prod.local`, etc.